recipes/awslimitchecker/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "awslimitchecker" %}
+{% set version = "10.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7610ac0fbe0817754e238099053a4ff4d3ea5260676094af86331891d6934a09
+
+build:
+  entry_points:
```

Comment:
Is `entry_points` correctly defined? Grayskull wasn't able to extract those.

Reference: https://github.com/jantman/awslimitchecker/blob/645e22dbe98841421c095837b621eb1188b732a2/setup.py#L81-L84

Reply:
Yeah those are usually defined as a dictionary, probably why grayskull couldn't extract

Reply:
Ahh ok, thanks!